api/spec/controllers/comments.spec.js

```diff
@@ -121,4 +134,243 @@ describe('/comments', () => {
       expect(response.body.token).toBe(undefined);
     });
   });
+
+  describe('Liking comments', () => {
+    it('should return an empty array when comment has no likes', async () => {
+      let post = new Post({
+        user_id: user_id,
+        comments: [],
+        message: 'posting howdy!',
+      });
+
+      await post.save();
+
+      let addingComment = await request(app)
+        .post('/comments')
+        .set('Authorization', `Bearer ${token}`)
+        .send({
+          user_id: user_id,
+          post_id: post._id,
+          message: 'commenting howdy!',
+        });
+
+      let response = await request(app)
+        .get('/posts')
+        .set('Authorization', `Bearer ${token}`);
+
+      let likeCount = 0;
+      console.log('response.body: ', response.body.posts[1].comments[0].likes);
+      response.body.posts[1].comments.forEach(
+        (comment) => (likeCount += comment.likes.length)
+      );
+      expect(likeCount).toEqual(0);
+    });
+
+    it('should have 1 like when the same user likes a comment twice', async () => {
+      let post = new Post({
+        user_id: user_id,
+        comments: [],
+        message: 'posting howdy!',
+      });
+
+      await post.save();
+
+      let addingComment = await request(app)
+        .post('/comments')
+        .set('Authorization', `Bearer ${token}`)
+        .send({
+          user_id: user_id,
+          post_id: post._id,
+          message: 'commenting howdy!',
+        });
+
+      let fetchingCommentForId = await request(app)
+        .get('/posts')
+        .set('Authorization', `Bearer ${token}`);
+
+      let likingComment = await request(app)
+        .patch('/comments/like')
+        .set('Authorization', `Bearer ${token}`)
+        .send({
+          _user_id: user_id,
+          _id: fetchingCommentForId.body.posts[1].comments[0]._id,
+        });
+
```

Comment:
Your naming of the requests seems a little confusing to me, I think it should be clear they are requests rather than comment objects as the naming implies